docs/config.md

```diff
@@ -46,3 +46,4 @@ Each service description is an object with the keys:
 
  * `authority` _String_. The domain name which the annotation service is associated with.
  * `grantToken` _String|null_. An OAuth grant token which the client can exchange for an access token in order to make authenticated requests to the service. If _null_, the user will only be able to read rather than create or modify annotations. (Default: _null_)
+ * `icon` _String|null_. The path to an image for the annotation service.
```

Comment:
We should add a note here about:

1. The recommended size/format. I think we agreed SVG suitable for a 16x16 image?
2. Where the icon will appear.